.releaserc.yml

```diff
@@ -0,0 +1,6 @@
+branch: master
+plugins:
+  - '@semantic-release/commit-analyzer'
```

Comment:
do we need this packages as dependency in package.json?

Reply:
Nope, semantic-release depends on these.